modules/installation-osp-control-compute-machines.adoc

```diff
@@ -14,18 +14,30 @@ ifeval::["{context}" == "installing-openstack-installer-ovs-dpdk"]
 :osp-sr-iov:
 endif::[]
 
-[id="installation-osp-control-compute-machines_{context}"]
-= Control plane and compute machines
+[id="installation-osp-control-machines_{context}"]
+= Control plane machines
 
 By default, the {product-title} installation process stands up three control
-plane and three compute machines.
+plane machines.
 
 Each machine requires:
 
 * An instance from the {rh-openstack} quota
 * A port from the {rh-openstack} quota
 * A flavor with at least 16 GB memory, 4 vCPUs, and 100 GB storage space
 
+[id="installation-osp-compute-machines_{context}"]
+= Compute machines
+
+By default, the {product-title} installation process stands up three compute
```

Comment:
I'm not sure if this sounds like an instance of anthropomorphism.
Per the the IBM Style Guide: 
_"Anthropomorphism": Do not assign human characteristics to inanimate objects, which are incapable of human behaviors and emotions. As much as possible, focus technical information on users and their actions, not on a product and its actions._ 
Suggest rewording `stands up `-> `creates/generates` in all the instances, but I'll let you and another peer reviewer make the call.

Reply:
Acked